src/code-server/install.sh

```diff
@@ -116,6 +120,18 @@ fi
 
 $(declare -p FLAGS)
 
+if [[ -f "$PASSWORDFILE" ]]; then
+	export PASSWORD="\$(cat '$PASSWORDFILE')"
```

Comment:
```suggestion
	export PASSWORD="\$(<"$PASSWORDFILE")"
```

Minor: [useless use of cat 😄](https://porkmail.org/era/unix/award) and `"` is fine within `$()`.

We don't have the same shell safety for inputs here (as with `declare -p`) but arguably if someone passes a file name with `"` in it then they might have other problems too.